ledger/sandbox/BUILD.bazel

```diff
@@ -273,75 +273,57 @@ NEXT_SERVERS = {
     },
 }
 
+# Server arguments that enable the append-only schema
+APPEND_ONLY_ARGS = [
+    "--enable-append-only-schema",
+]
+
+# Full conformance test
 server_conformance_test(
-    name = "next-conformance-test-static-time",
+    name = "next-conformance-test",
     lf_versions = [
         "default",
         "latest",
         "preview",
     ],
     server_args = [
-        "--static-time",
-    ],
-    servers = NEXT_SERVERS,
-    test_tool_args = [
-        "--open-world",
-        "--exclude=CommandDeduplicationIT",  # It's a KV ledger so it needs the KV variant
-        "--exclude=ClosedWorldIT",
-    ],
-)
-
-server_conformance_test(
-    name = "next-conformance-test-wall-clock",
-    server_args = [
-        "--wall-clock-time",
         "--max-deduplication-duration=PT5S",
-    ],
+    ] + APPEND_ONLY_ARGS,
     servers = NEXT_SERVERS,
     test_tool_args = [
         "--open-world",
-        "--additional=KVCommandDeduplicationIT",
+        "--additional=AppendOnlyCommandDeduplicationParallelIT",
+        "--additional=AppendOnlyCompletionDeduplicationInfoITCommandService",
+        "--additional=AppendOnlyCompletionDeduplicationInfoITCommandSubmissionService",
+        "--additional=AppendOnlyKVCommandDeduplicationIT",
         "--exclude=CommandDeduplicationIT",  # It's a KV ledger so it needs the KV variant
+        "--additional=ContractIdIT:RejectV0,ContractIdIT:AcceptSuffixedV1,ContractIdIT:AcceptNonSuffixedV1",
```

Comment:
Why do we run these here as well? Were these previously omitted? Or does it cover previous `next-conformance-test-contract-ids`?